src/Compilers/CSharp/Test/Semantic/Semantics/MethodTypeInferenceTests.cs

```diff
@@ -633,6 +635,155 @@ static void Main(string[] args)
                 Diagnostic(ErrorCode.ERR_BadArity, "Goo<>").WithArguments("Program.Goo<T, U>(T, U)", "method", "2"));
         }
 
+        [Fact]
+        public void Bug50782_2()
+        {
+            string source = @"
+#nullable enable
+
+interface IOperation<T> { }
+
+public class StringOperation : IOperation<string?> { }
+
+public class C {   
```

Comment:
nit: although this is test code, it would still be nice to follow our formatting conventions (ie. open brace on new line)